src/main.cpp

```diff
@@ -1059,6 +1074,8 @@ CAmount GetMinRelayFee(const CTransaction& tx, unsigned int nBytes, bool fAllowF
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
                         bool* pfMissingInputs, bool fRejectAbsurdFee)
 {
+    // FIXME: Overwinter txs check expiry height here?
```

Comment:
I guess we add the field in this PR, and then @arcalinea rebases her PR on top with the logic?

Reply:
Yes, that could work, since dependency is: activation logic <-- tx format <-- tx expity logic

Reply:
Added